setup.sh

```diff
@@ -78,10 +78,16 @@ if ! chmod +x "$tmp_file"; then
 fi
 printf "OK!\n"
 
-# Move the binary to /usr/local/bin and handle permission errors
-if ! mv "$tmp_file" /usr/local/bin/wd-41; then
-  echo "Failed to move the binary to /usr/local/bin/wd-41, see error above. Try running the script with sudo, or run 'mv $tmp_file <desired-position>'."
+# Move the binary to standard XDG location and handle permission errors
+INSTALL_DIR=$HOME/.local/bin
+# If run as 'sudo', install to /usr/local/bin for systemwide use
+if [ $EUID -eq 0 ]; then
```

Comment:
This didn't work on my machine, I suspect that `EUID` is a variable added in bash:
```bash
(zsh)❯ echo $EUID
1000
❯ sh
$ echo $EUID
(empty-string)
```

Doing it this way worked though:
```shell
if [ "$(id -u)" -eq 0 ]; then
	INSTALL_DIR=/usr/local/bin
fi
```